packages/components/src/Select/Option.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useRef } from 'react';
+import type { ListState } from 'react-stately';
+import type { Node } from '@react-types/shared';
+import { mergeProps, useOption } from 'react-aria';
+
+import { Box } from '../Box';
+import { useStyles } from '@marigold/system';
+
+interface OptionProps {
+  item: Node<unknown>;
+  state: ListState<unknown>;
+}
+
+export const Option = ({ item, state }: OptionProps) => {
+  const ref = useRef<HTMLLIElement>(null);
+  const { optionProps, isSelected, isFocused } = useOption(
+    {
+      key: item.key,
+    },
+    state,
+    ref
+  );
+  const selectedClassName = useStyles({
+    css: {
```

Comment:
Don't style stuff. Only layouting. Especially do *not* hardcode colors or spacing!